requirements.txt

```diff
@@ -1,5 +1,5 @@
 bs4==0.0.1
 requests==2.20.1
-scapy==2.4.0
```

Comment:
Why!?

Reply:
Correct this.

Reply:
Corrected now. I deleted it by mistake,got confused between scapy and scrapy.